assets/detail/ico-commits-last.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><g fill-rule="nonzero" fill="#666"><path d="M12 23.5C5.659 23.5.5 18.341.5 12S5.659.5 12 .5 23.5 5.659 23.5 12 18.341 23.5 12 23.5zm0-22C6.21 1.5 1.5 6.21 1.5 12S6.21 22.5 12 22.5 22.5 17.79 22.5 12 17.79 1.5 12 1.5z"/><path d="M12 12.5a.5.5 0 0 1-.5-.5V6a.5.5 0 0 1 1 0v4.793l2.146-2.146a.5.5 0 0 1 .707.707l-3 3A.5.5 0 0 1 12 12.5z"/><path d="M12 20.5c-4.687 0-8.5-3.813-8.5-8.5 0-4.687 3.813-8.5 8.5-8.5 4.687 0 8.5 3.813 8.5 8.5 0 4.687-3.813 8.5-8.5 8.5zm0-16c-4.136 0-7.5 3.364-7.5 7.5s3.364 7.5 7.5 7.5 7.5-3.364 7.5-7.5-3.364-7.5-7.5-7.5z"/></g></svg>
```

Comment:
Some of these SVGs have nice formatting (eg. `ico-changelog.svg`) while others are all on one line (eg. `ico-commits-last.svg`). Could you please make them consistent? Maybe format all of them nicely, as we can always minify them during the build.